### PR TITLE
LPD-19005 - Display restApplication field in the Data set table

### PR DESCRIPTION
**Pull request by juanjofgliferay**
## Task
https://liferay.atlassian.net/browse/LPD-19005

### CODE REVIEW

**Comment by juanjofgliferay** [comment]
ci:test:playwright

File: modules/test/playwright/tests/frontend-data-set-views-web/dataSets.spec.ts [hunk 1]
```diff
@@ -43,6 +44,36 @@ test('Assert table column labels', async ({dataSetsPage, page}) => {
...
+test('Assert table cell content', async ({dataSetsPage, page}) => {
+	const dataSetConfig = {
+		name: DEFAULT_LABEL.DATA_SET,
+		restApplication: '/data-set-manager/fields',
+		restEndpoint: '/',
+		restSchema: 'FDSField',
+	};
+
+	await dataSetsPage.goto();
+	await dataSetsPage.createDataSet(dataSetConfig);
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-data-set-views-web/dataSets.spec.ts:56]: We can now use creation and deletion data [with API](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-views-web/helpers/DataSetManagerApiHelpers.ts). It's much faster, less prone to flakiness, and easier to maintain compared to relying on UI.

**Review by markocikos — COMMENTED** [review]
@juanjofgliferay Nice addition of test! :+1: Only one improvement suggestion inline.

File: modules/test/playwright/tests/frontend-data-set-views-web/dataSets.spec.ts (hunk 1 above)
Comment (juanjofgliferay) [reply modules/test/playwright/tests/frontend-data-set-views-web/dataSets.spec.ts:56]: I've consider the creation of the data set via API, but it also makes sense testing the UI part.
Right now there are 3 tests that create/delete the dataset. We could refactor this and create just a couple of test that test each approach:
- everything via api (columns, content, actions)
- everything via UI (columns, content, actions)

**Review by markocikos — APPROVED** [review]
@juanjofgliferay LGTM! :sparkles:

**Comment by juanjofgliferay** [comment]
ci:forward

**Pull request closed without merging**